packages/menu/menu-item.md

```diff
@@ -29,14 +29,26 @@ import { MenuItem } from '@spectrum-web-components/menu';
 Menus are a collection of `<sp-menu-item>`s that can be modified via a `disabled` or `selected` attribute to represent an item in that state.
 
 ```html
-<sp-menu>
+<sp-menu selectable>
     <sp-menu-item>Active Menu Item</sp-menu-item>
     <sp-menu-item disabled>Disabled Menu Item</sp-menu-item>
     <sp-menu-item selected>Selected Menu Item</sp-menu-item>
 </sp-menu>
 ```
 
-### Value
+### Value slot
```

Comment:
Great addition! 💯  Shall the _icon_ `slot` also be documented here or it is considered as an `ActionButton` heritage?

Reply:
I think that will make sense to add in the upcoming Icon Picker work, yes!